daffodil-lib/src/main/scala/org/apache/daffodil/util/Cursor.scala

```diff
@@ -108,7 +108,7 @@ trait Cursor[AccessorType <: Accessor[AccessorType]] {
    * Like inspect, but only partially populates the accessor with data that can be computed without
    * side effects
    */
-  def inspectPure: Boolean
+  // def inspectPure: Boolean
```

Comment:
Remove comment. Was something fixed so inspectPure is no longer needed?

Reply:
Yes, inspectPure is no longer needed because pulling events no longer has side-effects of splicing things into the infoset. That's up to the caller of the inspect now (the Unparsers).

Reply:
Next commit removes all traces of inspectPure, including comments, etc.